probe/host/system_test.go

```diff
@@ -0,0 +1,38 @@
+package host_test
```

Comment:
- I didn't remove any tests, only added some
- You can't mock syscall.Uname, as it's only defined for GOOS=linux
- You can't really mock ReadFile, as the files you're reading (/proc) don't exist on Darwin, so the implementation would fail...